spec/aruba/fixtures/database_with_default_username_and_password.yml

```diff
@@ -0,0 +1,14 @@
+default: &default
+  username: postgres
+  password: ""
+  adapter: chronomodel
+  encoding: unicode
+  pool: 20
+  database: chronomodel_railsapp
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
```

Comment:
`git diff 40e0951..HEAD --stat` tells me this file was moved from `overwrite/config/database.yml`. So, no default database.yml anymore?

Reply:
I have mixed feelings about this. Maybe we should use sane defaults, that is do not provide username & password and use the defaults, and provide a way for the end user to specify username and password on the command line... WDYT?

Reply:
Sounds reasonable to me. :+1: